core/base/src/main/java/alluxio/security/authorization/Mode.java

```diff
@@ -185,6 +183,11 @@ public void setOtherBits(Bits bits) {
     mOtherBits = bits;
   }
 
+  /**
+   * @param u the owner bits to use
```

Comment:
here "bits" refer to POSIX permissions (rwx). I believe it could be more clear if you state them as "the owner permissions to use". The same values for group and other.